### ISSUE TITLE
How to reproduce numbers for API Bank & what are reward env vars used in paper?

### ISSUE THREAD

**Issue by sgdgp** [qiancheng0/ToolRL#45]
Hi,

Thanks for the the really interesting paper and open-sourced setup. I have two questions to reproduce numbers for API Bank reported in the paper when using Qwen2.5-3B-Instruct. I trained it using GRPO and the provided script for training.

- What are the settings to be used here to reproduce paper results? I kept all as `0` but is that desired?
```
export WITHLENGTH=0
export REFINEDREWARD=0
export COARSEREWARD=0
export STRICTMATCH=0
export CORRECTMAX1=0
export MAX1STEP30MAX3=0
export SCHEDULEREWARD=0
export SCHEDULELENGTH=0
```

- How do you choose the best epoch to use for evaluation? Is it from the validation score? Out of all the checkpoints saved at steps 15,30,45, 60, 75 and 90 (as per the save frequency and epoch count mentioned in paper) which epoch turned out to be best in the reported results?

**Comment by qiancheng0** [comment]
We use the validation set accuracy as heuristics for us to choose the final evaluated checkpoint during benchmarking.

**Comment by sgdgp** [comment]
Thanks, Could you please also tell about which environment variables are set to `1` for reproducing the numbers in `Table 2` of the paper? (More specifically was Dynamic Reward Scaling use for this table as the ablation showed it works better) ?

**Comment by qiancheng0** [comment]
Dynamic Reward Scaling is not used for the experiments in the main table, and it only serves as an ablation study.
